### ISSUE TITLE
Orders not displayed in Sales grid but appear in Last Orders on Dashboard

### ISSUE THREAD

**Issue by snswap828** [magento/magento2#5014]
Error in display orders
1. Order not display in Sell->Order 
2. Display in Dashboard  

![1](https://cloud.githubusercontent.com/assets/14049085/16034362/ff7463a8-322e-11e6-89c4-657b9bacd414.png)
![2](https://cloud.githubusercontent.com/assets/14049085/16034361/fef49524-322e-11e6-871a-a89629c11188.png)

**Comment by NadiyaS** [comment]
Hi @snswap828 ,
this behavior can not be reproducible on any Magento version because it is one of the most basic scenarios. So we need to know more details which could affect this issue. 
If you use additional extensions then please disable them, clean the cache and try to reproduce this bug again.
And also write Magento version you use.

**Comment by pboisvert** [comment]
@snswap828 please provide more info on version and steps to reproduce or else we will need to close this ticket.

**Comment by snswap828** [comment]
Magento version 2.0.5

when we login in admin panel, on dashboard it shows last 5 orders ,
but when we go to sell-> orders then it will display no item to display

On Fri, Jun 17, 2016 at 11:40 AM, Paul Boisvert notifications@github.com
wrote:

> @snswap828 https://github.com/snswap828 please provide more info on
> version and steps to reproduce or else we will need to close this ticket.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> https://github.com/magento/magento2/issues/5014#issuecomment-226688253,
> or mute the thread
> https://github.com/notifications/unsubscribe/ANZfPST-8Gu2VcDGhwA_f5lgoQ7FhpiVks5qMjpXgaJpZM4I1B_M
> .

## 

Thanks and Regards,

_Swapnil Nandanwar_

_Associate Developer_
M: +91-909-636-3612

**Comment by NadiyaS** [comment]
Hi @snswap828 ,
Seems like you have installed external extensions. Have you disabled all of them?
if you don't have any please provide us more details to reproduce this bug. All steps from the beginning. which setting did you change and so on.

**Comment by andimov** [comment]
@snswap828 
According to [contributor guide](http://devdocs.magento.com/guides/v2.1/contributor-guide/contributing.html), tickets without response for two weeks should be closed.
Please feel free to reopen if it's needed.

**Comment by shanevbg** [comment]
I know this issue was closed, but I have the exact same issue, ever since I upgraded from Magento 2.0 to 2.1 (and now 2.1.1), I haven't been able to see the orders in the order grid.  I've tried disabling extensions, etc.  I can get to the orders by either going to transactions and see them there, or the dashboard, but the order grid doesn't show any entries from after I updated to 2.1.  It is the div class="admin__old" if that is of any help.
